netlogo-gui/src/main/app/InterfaceTab.scala

```diff
@@ -31,6 +31,7 @@ class InterfaceTab(workspace: GUIWorkspace,
     // way so that only a vertical scrollbar is really needed - ST 7/13/04
     ScrollPaneConstants.VERTICAL_SCROLLBAR_ALWAYS,
     ScrollPaneConstants.HORIZONTAL_SCROLLBAR_AS_NEEDED)
+  scrollPane.setBorder(new javax.swing.border.EmptyBorder(0, 0, 0, 0))
```

Comment:
I should also point out that the Interface tab toolbar's height is inconsistent with the Info tab and the Code tab toolbars (on Linux with Nimbus LaF, anyway)